mobileChatExamples/androidChatExample/README.md

```diff
@@ -51,83 +56,159 @@ https://github.com/amazon-connect/amazon-connect-chat-ui-examples/assets/1439784
-  - If it exists, the module proceeds to fetch the chat transcript, allowing the user to continue from where they left off.
-  - If it does not exist, the module then checks for a `contactId`.
-
-- **Use of Contact ID:**
-  If a `contactId` exists, the module prompts the user to either restore the previous session or start a new chat.
-  - If the user chooses to restore, the module starts a new chat session with the existing `contactId`, creates a new participant connection, and then fetches the transcript.
-  - If the user opts for a new chat, the module deletes the stored `contactId` and `participantToken` from the storage, ensuring a fresh start. The chat begins with no prior context, emulating the start of a new conversation.
-
-      > There will be a new `initialContactId` when chat is rehydrated. The existing `contactId` will only be used as the `sourceContactId`. You may need to use the new `initialContactId` for the `CreateParticipantConnection` call or other APIs if you want to operate on the new contact.
-
-- **Deleting Stored Values:**
-  For users who opt to start a new chat, the module ensures that previous session identifiers are cleared. This action prevents any overlap or confusion between different chat sessions. By removing the `participantToken` and `contactId`, the ChatViewModel guarantees that the new chat session does not carry over any data or context from previous sessions.
-
-```mermaid
-flowchart TD
-    style D fill:#fc6b03 size:10
-    A[Start Chat] --> B{Check for participantToken}
-    B -->|Exists| J[Fetch Chat Transcript]
-    B -->|Does not exist| D{Check for contactId}
-    D -->|Exists| E{User Choice}
-    E -->|Restore| F[Use existing contactId]
-    E -->|New Chat| G[Delete stored contactId and participantToken]
-    F --> H[Create new participant connection]
-    G --> I[Start fresh chat session]
-    H --> J[Fetch Chat Transcript]
-    I --> J
-    J --> K[Continue where left off]
+## Implementation
+
+The first step is to call the `StartChatContact` API and pass the response details into the SDK’s `ChatSession` object. Here are some examples of how we would set this up in Kotlin. For reference, you can visit the `androidChatExample` demo within the Amazon Connect Chat UI Examples GitHub repository.
+
+### Configuring and Using `ChatSession` in Your Project
+
+The first step to leveraging the Amazon Connect Chat SDK after installation is to import the library into your file. Next, let's call the StartChatContact API and pass the response details into the SDK’s ChatSession object.  Here is an [example](TODO - Add link to UI Example) of how we would set this up in Kotlin. For reference, you can visit the [AndroidChatExample demo](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/mobileChatExamples/androidChatExample) within the [Amazon Connect Chat UI Examples](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master) GitHub repository.
```

Comment:
> For reference, you can visit the androidChatExample demo within the Amazon Connect Chat UI Examples GitHub repository.

This is a repeat of the end of the paragraph above.

Reply:
Yea, fixed it. Added some examples.